config/wdio/wdio.conf.js

```diff
@@ -71,30 +82,25 @@ const config = {
   ...locale && { locale },
   ...formFactor && { formFactor },
 
-  seleniumVersion: '3.14',
-  seleniumDocker: {
-    enabled: !ci,
-  },
-
   // Ignore deprecation warnings. When chrome supports /actions API we'll update to use those.
   deprecationWarnings: false,
 
   axe: {
     inject: true,
   },
 
+  terra: {
+    selector: '[data-terra-dev-site-content] *:first-child',
```

Comment:
Part of me wants to change this selector name, but i think that would be counter productive.

Reply:
Yeah, I don't really love this selector name referencing terra-dev-site, but wasn't sure what the best option would be. I think its okay enough since anyone can change it 🤷‍♀